prov/util/src/util_mem_hooks.c

```diff
@@ -160,7 +160,7 @@ static struct ofi_intercept intercepts[] = {
 extern void *__curbrk; /* in libc */
 #endif
 
-#if HAVE___CLEAR_CACHE
+#ifdef HAVE___CLEAR_CACHE
```

Comment:
The previous check required that HAVE__CLEAR_CACHE be set to non-zero.  This check now falls through as long as it's defined, regardless of the value.  Is this change correct in that HAVE__CLEAR_CACHE is either defined or not, and the value of it doesn't matter?